src/librawspeed/decompressors/LJpegPlain.cpp

```diff
@@ -259,9 +259,7 @@ void LJpegPlain::decodeN_X_Y() {
 
     if (X_S_F == 1) // will be optimized out
       for (uint32 i = 0; i < skipX; i++)
-        unroll_loop<N_COMP>([&](int i) {
-          ht[i]->decodeNext(bitStream);
-        });
+        unroll_loop<N_COMP>([&](int j) { ht[j]->decodeNext(bitStream); });
```

Comment:
I see a lot of [lambda-related clang-format bugs](https://llvm.org/bugs/buglist.cgi?quicksearch=clang-format+lambda), and i fail to find a config option that would change formatting to the suggested-one. So i guess this will stay as-is until it is fixed in clang-format no matter whether you create an issue here or not...